Module1/todo_dev.yaml

```diff
@@ -0,0 +1,55 @@
+# Comments are provided throughout this file to help you get started.
+# If you need more help, visit the Docker compose reference guide at
+# https://docs.docker.com/compose/compose-file/
+
+# Here the instructions define your application as a service called "server".
+# This service is built from the Dockerfile in the current directory.
+# You can add other services your application may depend on here, such as a
+# database or a cache. For examples, see the Awesome Compose repository:
+# https://github.com/docker/awesome-compose
+version: '3'
+
+services:
+  to_do_app:
+    image: module5_todo:dev
+    ports:
+      - "5000:5000"
+    env_file:
+      - ./.env
+    volumes:
+      - type: bind
+        source: ./todo_app
+        target: /app/todo_app
+# The commented out section below is an example of how to define a PostgreSQL
```

Comment:
Can we get rid of this commented out section now?

Reply:
comments removed